src/coreclr/ToolBox/superpmi/superpmi-shim-collector/superpmi-shim-collector.cpp

```diff
@@ -153,7 +153,7 @@ extern "C"
     return TRUE;
 }
 
-extern "C" DLLEXPORT void __stdcall jitStartup(ICorJitHost* host)
+extern "C" DLLEXPORT void jitStartup(ICorJitHost* host)
```

Comment:
@BruceForstall  is the __stdcall removal ok?

Reply:
Yes, it is ok. We preffer to just use default calling convention whenever possible.